components/list/list-item-generic-layout.js

```diff
@@ -56,16 +60,51 @@ class ListItemGenericLayout extends LitElement {
 		`;
 	}
 
+	constructor() {
+		super();
+		this._preventFocus = {
+			handleEvent(event) {
+				event.preventDefault();
+				// target content slot only for now - can add others later
+				// TODO findComposedAncestor should be replaced with event.path || event.composedPath() when supported
```

Comment:
I think `composedPath` should be usable.  We use it in a few places, usually to get at the real composed target.  It would have been polyfilled for IE11, and we won't be supporting IE11 very shortly.  That being said, I think what you have is ok as well.

Reply:
Will leave it for now with the todo